### PR TITLE
Switch to a new Artifactory proxy.

### PR DESCRIPTION
**Pull request by gkossakowski**
This one replaces a7 that we used previously.

### CODE REVIEW

**Comment by gkossakowski** [comment]
It's being validated here: https://jenkins-dbuild.typesafe.com:8499/job/Community-2.11.x-manual/132/console

Review by @lrytz

File: common-2.11.x.conf [hunk 1]
```diff
@@ -140,11 +140,11 @@ vars.base.extra.commands += "appendScalacOptions "${vars.scalac-opts}
 vars.ivyPat: ", [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]"
 options.resolvers: {
   0: "local"
-  1: "a7-maven: https://a7.typesafe.com:8082/artifactory/repo"
-  2: "a7-ivy: https://a7.typesafe.com:8082/artifactory/repo"${vars.ivyPat}
+  1: "a7-maven: https://proxy-ch.typesafe.com:8082/artifactory/repo"
+  2: "a7-ivy: https://proxy-ch.typesafe.com:8082/artifactory/repo"${vars.ivyPat}
```
Comment (lrytz) [inline common-2.11.x.conf]: the names are not updated, but i guess that doesn't matter

Comment (gkossakowski) [inline common-2.11.x.conf]: It doesn't but I'll update. Good catch.

**Comment by gkossakowski** [comment]
I fixed the names. The manual build seems to be resolved artifacts just fine so I'll merge it without waiting for the entire build to finish.

**Comment by lrytz** [comment]
lgtm

**Pull request merged**